packages/property-grid/src/components/PropertyGrid.scss

```diff
@@ -6,6 +6,10 @@
 @import "~@bentley/ui-core/lib/ui-core/style/index";
 @import "~@bentley/ui-core/lib/ui-core/text/index";
 
+.pcomponents-property-grid-with-unified-selection {
```

Comment:
where is this `pcomponents-` coming from?

Reply:
That's the class defined in the presentation-components class. It's what's leading to the extra scroll bars, so I'm overriding it.

Reply:
ok and i guess the class is coming from @bentley/presentation-components?